test/util/sqs.spec.js

```diff
@@ -0,0 +1,122 @@
+const path = require('path');
```

Comment:
Some results in the expect statements can be a little large. Should we move some of the data to a mock folder to reduce file size?

Reply:
No, that shouldn't be necessary here. A better thing to do is to generify your data going out, as well as your expected results going back in. Think about this:

```js
const sentEvent = { type: 'discover', ... };

// success test
aws.sqs.sendMessageBatch([sentEvent]).to.deep.equal(...);

// fail test
aws.sqs.sendMessageBatch([sentEvent]).to.throw(...) // or w/e